functions.R

```diff
@@ -312,9 +312,7 @@ compare_local_and_gdrive <- function(l_path, g_path){
   #' It is also useful because this version of modifiedTime does not change if someone were to edit the file in any way, 
   #' including renaming it to something else and back.  
   drive_rev_info <- googledrive::drive_reveal(g_path$files, what = "published")$revision_resource
-  drive_mtime <- as.POSIXct(sapply(
-    sapply(drive_rev_info, "[[", "modifiedTime"),
-    function(x) as.POSIXct(x, format = "%Y-%m-%dT%H:%M:%OSZ", tz = "GMT"), USE.NAMES = F))
```

Comment:
So you didn't need the date format specified? I'll see if this runs for me (as well as on some other objects).

Reply:
Ah, I see now that I chopped off the time with this change. I tried "1960-01-01" and "1960-01-02" for the origin on the outer call to as.POSIXct(), but couldn't seem to get the time component right.

Reply:
What error did you get from the original code?

Reply:
I got an error telling me that as.POSIXct() needed 'origin.'

Reply:
I can't reproduce this error. I'm curious which instance of as.POSIXct() is throwing the error for you. Typically, the origin is assumed as "1970-01-01" (not 1960), so I wonder if you're getting a different 'modifiedTime' date format than I am.

If you run lines 187+188 of the markdown (to create your_file_path and gdrive_test_dribble), and then the lines below, what do you get?

```
l_path  <- parse_local_path(your_file_path)
g_path <- parse_dribble(gdrive_test_dribble, l_path)
drive_rev_info <- googledrive::drive_reveal(g_path$files, what = "published")$revision_resource

# The character string of modified times from files in Gdrive
sapply(drive_rev_info, "[[", "modifiedTime")

# Datetimes converted to a numerical form
sapply(
    sapply(drive_rev_info, "[[", "modifiedTime"),
    function(x) as.POSIXct(x, format = "%Y-%m-%dT%H:%M:%OSZ", tz = "GMT"), USE.NAMES = F)
```

Reply:
![Capture](https://github.com/GeoffMayhew-NOAA-Affiliate/GoogleDriveTest/assets/11199899/5aecb704-2be1-4f02-97ff-e60d880df510)

Reply:
Adding the default origin solved the error. For some reason my computer assumed this origin whereas Phil's didn't.